lib/bloc/explorer/explorer_bloc.dart

```diff
@@ -350,6 +351,91 @@ class ExplorerBloc extends Bloc<ExplorerEvent, ExplorerState> {
     }
   }
 
+  Future<Account> _syncAccountUnstakes(Account account) async {
+    try {
+      // TODO(#542): use Unstakes instead of AddressBlocks
+      /// retrieve all Block Hashes
+      AddressBlocks? unstakes = (await explorer.address(
+              value: account.address,
+              // TODO(#542): get paginated Unstakes from unstake tab instead of AddressBlocks
+              tab: 'blocks') as PaginatedRequest<AddressBlocks?>)
+          .data;
+
+      if (unstakes != null) {
```

Comment:
`dart fix` detects this condition as unnecessary because `unstakes` can't be `null`

Reply:
unstakes can be null